extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -361,6 +409,29 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     });
   }
 
+  /**
+   * Handles Fluid-related messages dispatched from SafeFrame.
+   * @param {!JsonObject} data
+   */
+  receiveMessageForFluid(data) {
```

Comment:
make this private (and name ends in underscore)

Reply:
Done.